sample.env

```diff
@@ -16,6 +16,7 @@ DISCORD_TOKEN = "<discord_bot_token>"
 ## GOOGLE_SEARCH_API_KEY = "<google_api_key>" # allows internet searches and chats
 ## GOOGLE_SEARCH_ENGINE_ID = "<google_engine_id>" # allows internet searches and chats
 ## DEEPL_TOKEN = "<deepl_token>" # allows human language translations from DeepL API
+## GITHUB_TOKEN = "<github_access_token>" # allows indexing of GitHub repos
```

Comment:
There's no code that actually populates this github token into the runtime environment when it's loaded, can you add some to do that in EnvService?

Reply:
Got it, let me know if anything else is needed!